.changeset/fresh-wolves-behave.md

```diff
@@ -0,0 +1,10 @@
+---
+"@azure-tools/cadl-ranch": patch
+"@azure-tools/cadl-ranch-api": patch
+"@azure-tools/cadl-ranch-coverage-sdk": patch
+"@azure-tools/cadl-ranch-dashboard": patch
+"@azure-tools/cadl-ranch-expect": patch
+"@azure-tools/cadl-ranch-specs": patch
+---
+
+bump cadl deps
```

Comment:
As this will be part of the changelog in a readable way, we should try to make it a little more formal :)

Reply:
```suggestion
Move to CADL 0.39 as a baseline
```